trunk/includes/pardot-forms-widget-class.php

```diff
@@ -184,15 +184,15 @@ function __construct()
 		/**
 		 * Call the WP_Widget parent constructor.
 		 */
-		parent::__construct('pardot-forms', __('Pardot Forms', 'pardot'), $widget_ops, $control_ops);
+		parent::__construct('pardot-forms', __('Account Engagement Forms', 'pardot'), $widget_ops, $control_ops);
 	}
 	/*
 
 	*/
 	/**
-	 * Displays Pardot forms for Widget HTML on front end of site.
+	 * Displays Account Engagement forms for Widget HTML on front end of site.
 	 *
-	 * Echos form defined in Pardot account and specified for this widget.  Can display as IFrame or as inline HTML the
+	 * Echos form defined in Account Engagement business unit and specified for this widget.  Can display as iFrame or as inline HTML the
```

Comment:
I wouldn't use the `iFrame` capitalization pattern, as that's kind of in Apple's domain. I think technically all lowercase is correct: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe